reactor-core/src/test/java/reactor/core/publisher/FluxRetryWhenTest.java

```diff
@@ -21,9 +21,9 @@
 import java.util.ArrayList;
```

Comment:
Can we have a test where `Retry.Builder` is reused? Just to make sure that it is stateless and reusable